lesson3/tasks/2Task.js

```diff
@@ -0,0 +1,20 @@
+function loadUsers(path) {
+    return fetch(path)
+      .then(response => {
+        if (!response.ok) {
+          throw new Error('Network response was not ok.');
+        }
+  
+        return response.json();
+      })
+      .then(users => {
+        users.forEach(u => {
```

Comment:
I'd recommend renaming variable `u` to `user`. So this will be more intuitive of what a variable stands for. Otherwise looks good.